readme.md

```diff
@@ -217,6 +217,14 @@ Yields, in [hast][] (**note**: the `pre` and `language-js` class are normal
 }
 ```
 
+### `all(h, parent)`
+
+Helper function for writing custom handlers passed to `options.handlers`.
+
+### `one(h, node, parent)`
+
+Helper function for writing custom handlers passed to `options.handlers`.
```

Comment:
```suggestion
Helper function for writing custom handlers passed to `options.handlers`.
Pass it `h`, a `node`, and its `parent` (mdast) and it will turn `node` into
hast content.
```